README.md

```diff
@@ -63,7 +63,7 @@ have it assigned to yourself.
 
 ## Contributing
 
-Check out our [contributing guidelines](docs/CONTRIBUTING.md).
+Check out our [contributing guidelines](docs/development/CONTRIBUTING.md).
```

Comment:
The document won't be picked up from here by GitHub. I suggest leaving it there and creating new file in `development` directory.

Reply:
To be honest, I'd drop this commit all together and just add this single new markdown file into this new directory.

Reply:
What do you mean by `won't be picked up`? I see that the link works.

Reply:
https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file

Reply:
done.